### PR TITLE
Created code samples for setup intent

### PR DESCRIPTION
**Pull request by monica-m-ps**
Created and tested code samples for setup intent. 

What's pending?
- PHP code samples. 

Ticket for updating PHP SDK
https://paymentsense.atlassian.net/browse/PPPP-413

Created code samples for the following scenarios:

- Create setup intent
- Retrieve setup intent
- Charge card
- Create payment intent with setup intent ID.

### CODE REVIEW

File: setup-intent/cs/create-payment-intent.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+var paymentIntentsClient = new PaymentIntentsClient(new HttpClient(), new ApiKeyClientAuthorization("sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ"));
+var result = await paymentIntentsClient.CreatePaymentIntentAsync(new CreatePaymentIntentRequest
+{
+    Amount = new()
+    {
+        Value = 1000,
+        CurrencyCode = "GBP"
+    },
+    Reference = "Dojo Cafe | Table no.4.",
+    SetupIntentId = "si_<secret-key>",
```
Comment (AndreiMihalciuc) [inline setup-intent/cs/create-payment-intent.cs:15]: should be consistent with the rest of examples, e.g. below we have `"<Setup intent ID>"`

File: setup-intent/curl/create-payment-intent.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    --header 'Content-Type: application/json' \
+    --header 'Authorization: Basic sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ' \
+    --header 'Version: 2022-04-07' \
+    --data '{
+        "amount": {
+            "value": 1000,
+            "currencyCode": "GBP"
+        },
+        "reference": "Dojo Cafe | Table no. 4.",
+        "setupIntentId": "si_sandbox_6zrBhAXpYE2Q7QZPFB1flQ"
```
Comment (AndreiMihalciuc) [inline setup-intent/curl/create-payment-intent.sh:17]: replace with a generic value description as above

File: setup-intent/powershell/create-payment-intent.ps1 [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        "Authorization" = "Basic $publicSandboxKey"
+    } `
+    -ContentType 'application/json' `
+    -Body '{
+        "amount": {
+            "value": 1000,
+            "currencyCode": "GBP"
+        },
+        "reference": "Dojo Cafe | Table no. 4.",
+        "setupIntentId": "si_sandbox_6zrBhAXpYE2Q7QZPFB1flQ"
```
Comment (AndreiMihalciuc) [inline setup-intent/powershell/create-payment-intent.ps1:22]: same please

File: setup-intent/python/create-payment-intent.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+import json
+
+conn = http.client.HTTPSConnection("api.dojo.tech")
+payload = json.dumps({
+    "amount": {
+        "value": 1000,
+        "currencyCode": "GBP"
+    },
+    "reference": "Dojo Cafe | Table no. 4.",
+    "setupIntentId": "si_sandbox_6zrBhAXpYE2Q7QZPFB1flQ"
```
Comment (AndreiMihalciuc) [inline setup-intent/python/create-payment-intent.py:16]: same

**Review by aaronforinton-dojo — APPROVED** [review]

**Review by cameliastancu — APPROVED** [review]

File: setup-intent/cs/create-payment-intent.cs (hunk 1 above)
Comment (monica-m-ps) [reply setup-intent/cs/create-payment-intent.cs]: Done.

File: setup-intent/curl/create-payment-intent.sh (hunk 2 above)
Comment (monica-m-ps) [reply setup-intent/curl/create-payment-intent.sh]: Done

**Review by GabrielBoceanuDojo — APPROVED** [review]

File: setup-intent/cs/retrieve.cs [hunk 5]
```diff
@@ -0,0 +1,8 @@
+// The sandbox API key passed in 'ApiKeyClientAuthorization' is public.
+// Don't submit any personally identifiable information in any requests made with this key.
+// Sign in to developer.dojo.tech to create your own private sandbox key and use that instead
+// for secure testing.
+
+var setupIntentsClient = new SetupIntentsClient(new HttpClient(), new ApiKeyClientAuthorization("sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ"));
+var result = await setupIntentsClient.GetByIdAsync("<Setup intent ID>");
+Console.Write(result.Status);
```
Comment (RomanKondratyev) [inline setup-intent/cs/retrieve.cs:8]: WriteLine?

**Review by RomanKondratyev — APPROVED** [review]

File: setup-intent/cs/retrieve.cs (hunk 5 above)
Comment (monica-m-ps) [reply setup-intent/cs/retrieve.cs]: Done. Thanks

**Pull request merged**